### PR TITLE
Bugfix bot 1557 handling emails with accents

### PR DESCRIPTION
**Pull request by tchevily**
As swiftmailer uses RFC2822 standard, we should use the same standard in user form.

Reference : BOT-1557

### CODE REVIEW

**Comment by tchevily** [comment]
> If the problem is on the swiftmailer part. Couldn't just an update of the version of the library (https://github.com/CanalTP/NMM/blob/master/composer.json#L145) to version v3.2.6 for example just don't do the trick ?

This requirement comes form SamEcoreUserManagerBundle and it should be handled inside this bundle.
The swiftmailer Bundle could be used by other bundles and upgrade could create regression.

File: Resources/translations/FOSUserBundle.fr.yml [hunk 1]
```diff
@@ -112,3 +112,5 @@ form:
     password_confirmation: "Vérification :"
     new_password: "Nouveau mot de passe :"
     new_password_confirmation: "Vérification :"
+
+"This email address does not comply with RFC 2822" : "Cette adresse e-mail n'est pas conforme aux normes RFC 2822"
```
Comment (Fahmus) [inline Resources/translations/FOSUserBundle.fr.yml]: useless?

**Comment by Fahmus** [comment]
> If the problem is on the swiftmailer part. Couldn't just an update of the version of the library (https://github.com/CanalTP/NMM/blob/master/composer.json#L145) to version v3.2.6 for example just don't do the trick ?

Version 6.1.0: added support for non-ASCII email addresses
https://github.com/swiftmailer/swiftmailer/blob/master/CHANGES#L52

File: Resources/translations/FOSUserBundle.fr.yml (hunk 1 above)
Comment (pthegner) [reply Resources/translations/FOSUserBundle.fr.yml]: I don't think it's useless, just that it not really fit with the other trad key

**Pull request merged**